README.md

```diff
@@ -106,7 +105,7 @@ Once you're ready to learn more about **Stretch AI**, you can try out the variet
 
 You can use an LLM to provide free-form text input to the pick and place demo with the `--use_llm` command line argument.
 
-Running the following command will first download an open LLM model. Currently, the default model is [Qwen2.5-3B-Instruct](https://huggingface.co/Qwen/Qwen2.5-3B-Instruct). Running this command downloads ~10GB of data, so you'll need to be patient.
+Running the following command will first download an open LLM model. Currently, the default model is [Qwen2.5-3B-Instruct](https://huggingface.co/Qwen/Qwen2.5-3B-Instruct). Running this command downloads ~10GB of data. Using an ethernet cable instead of Wifi is recommended.
```

Comment:
I don't like this suggestion, it's awkward to plug in ethernet

Reply:
It's awkward, but effective. The robot-side docker was at 10% after 30min, and then finished in 10 min after I plugged in the cable.

Reply:
To be clear, I'm not saying keep it plugged in for the demo. Just the download of the 10GB.